roles/pulp/defaults/main.yml

```diff
@@ -5,7 +5,6 @@ pulp_config_dir: '/etc/pulp'
 # This var intentionally not advertised to users, because there is no
 # foreseeable need for them to change the filename, only pulp_config_dir.
 pulp_settings_file: '{{ pulp_config_dir }}/settings.py'
-pulp_default_admin_password: ''
```

Comment:
Does it make sense to add a `.removal` notice for these two vars now being required?

Reply:
IIRC, my research last week was that the empty string default would fail mid-play. So it's not really a removal of any functionality. I suppose that users could be comparing to the empty string in their own playbook or roles, but it seems very unlikely.